src/main/java/net/snowflake/ingest/utils/ErrorCode.java

```diff
@@ -36,7 +36,10 @@ public enum ErrorCode {
   CHANNEL_STATUS_INVALID("0028"),
   UNSUPPORTED_DATA_TYPE("0029"),
   INVALID_VALUE_ROW("0030"),
-  MAX_ROW_SIZE_EXCEEDED("0031");
+  MAX_ROW_SIZE_EXCEEDED("0031"),
+  MAKE_URI_FAILURE("0032"),
+  OAUTH_REFRESH_TOKEN_ERROR("0033"),
+  INVALID_CONFIG_PARAMETER("0034");
```

Comment:
I don't the corresponding error messages being added, did I miss anything?

Reply:
Used [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/2b7d20b3aa8250b51ef1c558f395dfd64b048c42/src/main/java/net/snowflake/ingest/utils/Utils.java#L128)

Reply:
@sfc-gh-alhuang Translations for these errors are missing, see https://github.com/snowflakedb/snowflake-ingest-java/blob/master/src/main/resources/net/snowflake/ingest/ingest_error_messages.properties#L36.

Reply:
@sfc-gh-lsembera Thanks for the hint! I'll send a PR to add the error messages.